02 Algorithm Reference/03 Handling Data/11 Delistings

```diff
@@ -0,0 +1,27 @@
+<meta name="tag" content="data event handlers"/>
+<meta name="tag" content="delistings event"/>
+
+<p>
+    $[Delisting,T:QuantConnect.Data.Market.Delisting] events provide notice an asset is no longer trading on the exchange.
```

Comment:
"A delisting warning is issued the final trading day for a stock delisting event to give you algorithm time to gracefully exit out of positions before forced termination"

Reply:
```suggestion
    $[Delisting,T:QuantConnect.Data.Market.Delisting] events provide notice an asset is no longer trading on the exchange.
```